__tests__/users.test.js

```diff
@@ -0,0 +1,101 @@
+const pool = require('../lib/utils/pool');
+const setup = require('../data/setup');
+const request = require('supertest');
+const app = require('../lib/app');
+const UserService = require('../lib/services/UserService');
+
```

Comment:
This file is named user.test.js but contains tests for both our users and posts. We might split those into two test files